internal/reader/ccip.go

```diff
@@ -412,6 +411,7 @@ func (r *CCIPChainReader) getSourceChainsConfig(
 				ctx,
 				consts.ContractNameOffRamp,
 				consts.MethodNameGetSourceChainConfig,
+				primitives.Finalized,
```

Comment:
New input param introduced. It represents the level of confidence of the data asked

Reply:
Interesting. I don't think this particular call needs to be finalized though

Reply:
Doesn't need to be finalized.